src/_posts/databases/postgresql/2000-01-01-accessing.md

```diff
@@ -0,0 +1,238 @@
+   ```bash
+   -----> Starting container one-off-7872  Done in 0.479 seconds
+   -----> Connecting to container [one-off-7872]...  
+   -----> Process 'pgsql-console my_app_4553' is starting...  
+
+   Hello from bashrc.
+   ---> Download and extract the database CLI
+   ---> Database CLI installed:
+   psql (PostgreSQL) 14.6
+   psql (14.6, server 14.8 (Debian 14.8-1.pgdg110+1))
+   SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+   Type "help" for help.
+
+   my_app_4553=>
+   ```
+3. To exit, type the following command from the PostgreSQL® console, or use the
+   keyboard shortcut `CTRL`+`D`:
+   ```bash
+   \q
+   ```
+
+
+## Using a One-Off
+
+1. [Start a one-off container]({% post_url platform/app/2000-01-01-tasks %})
+2. From the command line, use our `dbclient-fetcher` helper to download and
+   install a bunch of PostgreSQL® tools:
+   ```bash
+   dbclient-fetcher pgsql
+   ```
+   The output should look like this:
+   ```bash
+   ---> Download and extract the database CLI
+   ---> Database CLI installed:
+   psql (PostgreSQL) 14.6
+   ```
+3. If you ever need a specific version, just add it as a second parameter:
+   ```bash
+   dbclient-fetcher pgsql 13
+   ```
+   The output should look like this:
+   ```bash
+   ---> Download and extract the database CLI
+   ---> Database CLI installed:
+   psql (PostgreSQL) 13.11
+   ```
+
+The helper installs the following tools:
+- `psql`
+- `pg_basebackup`
+- `pg_controldata`
+- `pg_dump`
+- `pg_isready`
+- `pg_recvlogical`
+- `pg_restore`
+- `pg_test_fsync`
+- `pg_upgrade`
+- `pg_archivecleanup`
+- `pg_config`
+- `pg_ctl`
+- `pg_dumpall`
+- `pg_receivewal`
+- `pg_resetwal`
+- `pg_rewind`
+- `pg_test_timing`
+- `pg_waldump`
+
+
+## Using Third Party Tools
+
+By default, and for security reasons, your Scalingo for PostgreSQL® database is
+not directly accessible from the Internet.
+
+To access your database remotely you first need to [make it reachable over the
+Internet](#making-the-database-reachable-over-internet):
+- either locally, on your computer, via an [encrypted tunnel](#setting-up-an-encrypted-tunnel)
+- or from any location, by [enforcing TLS connection]({% post_url databases/postgresql/2000-01-01-connecting %}#enforcing-tls-connection)
+  and [enabling direct Internet access](#enabling-direct-access-over-internet).
+
+Once a secured connection has been established, you should be able to connect
+to your database with your tool of choice. If you don't have one yet, we
+suggest you to [take a look at pgAdmin](#using-pgadmin).
+
+###  Making the Database Reachable Over Internet
+
+#### Using Our Command Line Tool
+
+Our Command Line tool provides a `db-tunnel` command that creates an SSH
+encrypted tunnel, allowing remote access to your Scalingo for PostgreSQL®
+addon.
+
+This tunnel does not provide any additional security layer. We only verify that
+your public key is registered on our platform. It acts as an encrypted bridge
```

Comment:
issue: I find it disappointing regarding the security in place. I would rephrase in a more positive way. The use of negative words ("does not") or limiting word ("only") in this context makes me feel disappointed when reading the sentence. Whereas the security in place is good.

Reply:
Done in https://github.com/Scalingo/documentation/pull/2389/commits/b672481fa4f6324e041dc3b7d3293acd094b9c46